controllers/etcdcluster_controller.go

```diff
@@ -1025,6 +1048,14 @@ func nextOutdatedPeer(cluster *etcdv1alpha1.EtcdCluster, peers *etcdv1alpha1.Etc
 	return nil
 }
 
+func scheduleMapKeyFor(cluster *etcdv1alpha1.EtcdCluster) string {
+	return string(cluster.UID)
```

Comment:
nit: it might be worth making this `string(cluster.UID) + defrag`.

Just so it's clear what this key does and to stop us accidentally overlapping keys if we add more cron jobs in the future